sink/signalfx.go

```diff
@@ -136,8 +136,10 @@ func (s *SignalFx) Send(ctx context.Context, m *blip.Metrics) error {
 
 			// Convert Blip metric type to SFX metric type
 			switch metrics[i].Type {
-			case blip.COUNTER:
+			case blip.CUMULATIVE_COUNTER:
 				dp[n] = sfxclient.CumulativeF(name, dim, metrics[i].Value)
+			case blip.DELTA_COUNTER:
+				dp[n] = sfxclient.Counter(name, dim, int64(metrics[i].Value))
```

Comment:
`int64(float64)` is going to truncate the value (rather than rounding it), It probably doesn't matter in this case.

Reply:
Yeah shouldn't matter because they're really integers from MySQL. Blip uses float64 because that's more flexible overall for monitoring. I.e. if a tool uses int and then needs float, it's not really possible. But if a tool uses float, that works for the rare floats and ints.